content/en/docs/setup/learning-environment/minikube.md

```diff
@@ -204,7 +204,7 @@ plugins.
 * hyperv ([driver installation](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperv-driver))
 Note that the IP below is dynamic and can change. It can be retrieved with `minikube ip`.
 * vmware ([driver installation](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#vmware-unified-driver)) (VMware unified driver)
-* none (Runs the Kubernetes components on the host and not in a VM. It is not recommended to run the none driver on personal workstations. Using this driver requires Docker ([docker install](https://docs.docker.com/install/linux/docker-ce/ubuntu/)) and a Linux environment)
+* none (Runs the Kubernetes components on the host and not in a VM. It is not recommended to run the none driver on personal workstations. Using this driver requires Docker ([docker install](https://docs.docker.com/install/linux/docker-ce/ubuntu/)) and a Linux environment. Non docker runtimes are not supported with none driver. Docker is the only supported container runtime for none.)
```

Comment:
Sorry about the wait.

```suggestion
* none (Runs the Kubernetes components on the host and not in a virtual machine. You need to be running Linux and to have {{< glossary_tooltip term_id="docker" >}} installed. The Kubernetes components run as privileged containers and will have side effects outside of the Minikube environment, so the `none` driver is not recommended for personal workstations.)
```

Alternatively, use this:
```markdown
 * none (Runs the Kubernetes components on the host and not in a virtual machine. You need to be running Linux and to have {{< glossary_tooltip term_id="docker" >}} installed. 

{{< caution >}}
The Kubernetes components run as privileged containers that have side effects outside of the Minikube environment, so the `none` driver is not recommended for personal workstations.
{{< /caution >}}
```

Reply:
sure